docs/codeql/reusables/beta-note-package-management.rst

```diff
@@ -2,4 +2,4 @@
 
     Note
 
-    The CodeQL package management functionality, including CodeQL packs, is currently available as a beta release and is subject to change. During the beta release, CodeQL packs are available only using GitHub Packages - the GitHub Container registry. To use this beta functionality, install version 2.6.0 or higher of the CodeQL CLI bundle from: https://github.com/github/codeql-action/releases.
\ No newline at end of file
+    The CodeQL package management functionality, including CodeQL packs, is currently available as a beta release and is subject to change. During the beta release, CodeQL packs are available only using GitHub Packages - the GitHub Container registry. To use this beta functionality, install the latest version of the CodeQL CLI bundle from: https://github.com/github/codeql-action/releases.
```

Comment:
Maybe "latest version" is misleading, but equally, we don't want people on old versions of the CodeQL CLI bundle getting frustrated because it doesn't work, or doesn't work with the GHES Container registry.

We could give a minimum version number?

Reply:
Sure. I think that makes the most sense.